tests/installation/welcome.pm

```diff
@@ -113,6 +135,7 @@ sub run {
     }
 
     assert_screen 'languagepicked';
+    check_keyboard_layout;
```

Comment:
Would it be possible for you to make that an explicit test module rather than "hide" it in "welcome"?